### ISSUE TITLE
How do we use the variant of your current models : Ours -im-l2z and Ours-im-pc2z

### ISSUE THREAD

**Issue by prashant030591** [ChrisWu1997/Multimodal-Shape-Completion#1]
Hi  @ChrisWu1997 

Can you please tell me how does one use the 2 variants of your models: Ours -im-l2z and Ours-im-pc2z.
Is there any flag or different models made for it, or does one need to manually change the model for running these variants,

I tried searching for some ready-made configuration that can help run these but could not find anyone.

Thanks,
Prashant

**Comment by ChrisWu1997** [comment]
Thanks for taking interest in our paper. Currently the code doesn't support the 2 variants of our model. If you are not in a hurry, I'll update the code to support the 2 variants within a week and inform you once it's done.

**Comment by prashant030591** [comment]
Hi Rundi,

Thanks for your prompt reply.Really appreciate it.

Actually we ar implementing you paper as a baseline for our work and we
were a little short ot time as the conference deadline is near.

Would really request to help us with your code as your paper as soon as
possible because your paper is a very excellent baseline for ours and we
really don't want to miss including it.


Thanks,
Prashant

**Comment by ChrisWu1997** [comment]
I see. In this case, I can send you the code for our 2 variants independently (not integrated into this repo). On which dataset do you need? Or it doesn't matter if you are using your own dataset?

**Comment by prashant030591** [comment]
Yes, Actually I am using my own custom dataset which I have setup in the
CSV file and in the 2 folders :  ShapenetPointCloud and the other one.
You can share it with me here.

**Comment by ChrisWu1997** [comment]
Two code folders attached. Feel free to contact me if there are any questions. 


> 2020年8月14日 下午9:44，PRASHANT KUMAR <notifications@github.com> 写道：
> 
> 
> Yes, Actually I am using my own custom dataset which I have setup in the
> CSV file and in the 2 folders : ShapenetPointCloud and the other one.
> You can share it with me here.
>

**Comment by prashant030591** [comment]
Thanks, Chris for this early resolution

**Comment by ChrisWu1997** [comment]
I just resend the code directly to your gmail, please check:)

**Comment by prashant030591** [comment]
Got it, thanks to you, Chris 👍

**Comment by prashant030591** [comment]
Hi Chris,

In the files sent , there is a file common.py that mentions.
 self.ae_modelpath = "/mnt/disk4/wurundi/multimodal-pc-completion/3depn-table-pointnetAE-small-2K/model/ckpt_epoch3000.pth"

Does this models refer to the ae or the have trained using the complete Point Clouds?
From your paper, it appears that you use the VAE trained using completely PC in the 2 variants.

Needed a clarification as to whether the line 👍 
 self.ae_modelpath = "/mnt/disk4/wurundi/multimodal-pc-completion/3depn-table-pointnetAE-small-2K/model/ckpt_epoch3000.pth"

refers to the AE or the VAE trained previously.

-Prashant

**Comment by ChrisWu1997** [comment]
It refers to the AE pretrained using the complete point cloud. There’s no pretrained VAE involved in our 2 variants. I’ll double check the paper/supp later.
